aspnetcore/release-notes/aspnetcore-3.0.md

```diff
@@ -0,0 +1,501 @@
+---
+title: What's new in ASP.NET Core 3.0
+author: rick-anderson
+description: Learn about the new features in ASP.NET Core 3.0.
+ms.author: riande
+ms.custom: mvc
+ms.date: 09/19/2019
+uid: aspnetcore-3.0
+---
+# What's new in ASP.NET Core 3.0
+
+This article highlights the most significant changes in ASP.NET Core 3.0 with links to relevant documentation.
+
+## Blazor
+
+Blazor is a new framework in ASP.NET Core for building interactive client-side web UI with .NET:
+
+* Create rich interactive UIs using C# instead of JavaScript.
+* Share server-side and client-side app logic written in .NET.
+* Render the UI as HTML and CSS for wide browser support, including mobile browsers.
+
+Using .NET for client-side web development offers the following advantages:
```

Comment:
I think we can remove this list on the benefits of using .NET, as it is primarily targeted at developers that are new to .NET, which really isn't the audience for this article.